webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+
+            <h2>Basic aggregations</h2>
+
+            <p>Using the <code>_facets</code> parameter allows you to run simple aggregations. You can pass it a field name (see the <a href="{{ url('documentation:supersearch_api') }}#section-list-of-fields">List of fields</a>), and it will count the terms of that field in all the documents of the dataset. It will then return them sorted by descending count. </p>
+
+            <div class="example">
+                <p>Example</p>
+                <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?_facets=signature</span></code></pre>
+            </div>
+
+            <p>This will return the list of signatures that appear the most in all crash reports for the last week, sorted by descending count. </p>
+
+            <p>The number of results in an aggregation is 50 by default. If you want to change it, you can use the <code>_facets_size</code> parameter. Note that there is no way to paginate over aggregation results, and that the bigger the <code>_facets_size</code> the longer the request will take. </p>
+
+            <h2>Nested aggregations</h2>
+
+            <p>Let's say you want to get the count of platforms per product. To do that, you would use a nested aggregation: aggregate on products, and then for each product aggregate on platforms. Here's what that translates to in Super Search: </p>
+
+            <div class="example">
+                <p>Example</p>
+                <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?_aggs.product=platform</span></code></pre>
+            </div>
+
+            <h2>Special aggregations</h2>
+
+            <div class="examples">
+            <h1 id="examples">Examples</h1>
+
+            <h2>List of signatures</h2>
+
+            <h3>Goal</h3>
+            <p>Get the top 1000 signatures for Firefox 2.0. </p>
+
+            <h3>Solution</h3>
+            <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?product=Firefox&amp;version=2.0&amp;_facets=signature&amp;_facets_size=1000</span></code></pre>
+
+            <h3>Tips</h3>
+            <p>The <code>_facets_size</code> parameter allows you to set the number of results in all aggregations. Setting too big a number will of course make the request take longer. Note that there are no ways of paginating over results of an aggregation. </p>
+
+            <h2>Time range</h2>
+
+            <h3>Goal</h3>
+            <p>Get the first 100 crash reports for Firefox between Jan 1st, 2000 and Feb 1st, 2000. </p>
```

Comment:
Would be cool if this example used some real dates. E.g. Current month's 1st date and last months first date.